src/components/NotFound.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
```

Comment:
In that file I would suggest to declare NoContent components first and then declare NotFound component. I twill be easier to read

Reply:
Implemented